Index: freewnn-1.1.1~a021+cvs20100325/Wnn/jutil/wddel.c
===================================================================
--- freewnn-1.1.1~a021+cvs20100325.orig/Wnn/jutil/wddel.c	2012-05-31 22:00:49.079112665 +0900
+++ freewnn-1.1.1~a021+cvs20100325/Wnn/jutil/wddel.c	2012-05-31 22:01:14.151141002 +0900
@@ -95,7 +95,7 @@
 static void
 err ()
 {
-  printf ((char *) wnn_perror ());
+  printf ("%s", (char *) wnn_perror ());
   printf ("\n bye.\n");
   exit (1);
 }
Index: freewnn-1.1.1~a021+cvs20100325/Wnn/jutil/wdreg.c
===================================================================
--- freewnn-1.1.1~a021+cvs20100325.orig/Wnn/jutil/wdreg.c	2012-05-31 22:00:49.079112665 +0900
+++ freewnn-1.1.1~a021+cvs20100325/Wnn/jutil/wdreg.c	2012-05-31 22:01:14.151141002 +0900
@@ -95,7 +95,7 @@
 static void
 err ()
 {
-  printf ((char *) wnn_perror ());
+  printf ("%s", (char *) wnn_perror ());
   printf ("\n bye.\n");
   exit (1);
 }
Index: freewnn-1.1.1~a021+cvs20100325/Wnn/uum/select_ele.c
===================================================================
--- freewnn-1.1.1~a021+cvs20100325.orig/Wnn/uum/select_ele.c	2012-05-31 22:00:49.079112665 +0900
+++ freewnn-1.1.1~a021+cvs20100325/Wnn/uum/select_ele.c	2012-05-31 22:03:22.683286361 +0900
@@ -816,7 +816,7 @@
   /*
      sprintf(buf + strlen(buf), "」を削除します。(Y/N)");
    */
-  sprintf (buf + strlen (buf), MSG_GET (24));
+  sprintf ("%s", buf + strlen (buf), MSG_GET (24));
   if (yes_or_no (buf) == 1)
     {
       if (jl_word_delete (bun_data_, word_searched[lc].dic_no, word_searched[lc].serial) == -1)
@@ -852,14 +852,14 @@
       /*
          sprintf(buf + strlen(buf), "」の使用を再開します。(Y/N)");
        */
-      sprintf (buf + strlen (buf), MSG_GET (32));
+      sprintf ("%s", buf + strlen (buf), MSG_GET (32));
     }
   else
     {
       /*
          sprintf(buf + strlen(buf), "」の使用を中止します。(Y/N)");
        */
-      sprintf (buf + strlen (buf), MSG_GET (26));
+      sprintf ("%s", buf + strlen (buf), MSG_GET (26));
     }
   if (yes_or_no (buf) == 1)
     {
Index: freewnn-1.1.1~a021+cvs20100325/Wnn/uum/inspect.c
===================================================================
--- freewnn-1.1.1~a021+cvs20100325.orig/Wnn/uum/inspect.c	2002-03-30 10:45:41.000000000 +0900
+++ freewnn-1.1.1~a021+cvs20100325/Wnn/uum/inspect.c	2012-05-31 22:07:37.735575236 +0900
@@ -213,7 +213,7 @@
   sStrcpy (buf + strlen (buf), w_buf);
   set_escape_code (buf);
   strcat (buf, " ");
-  sprintf (buf + strlen (buf), MSG_GET (20));
+  sprintf ("%s", buf + strlen (buf), MSG_GET (20));
   /*
      sprintf(buf + strlen(buf), "を削除しますか?(Y/N)");
    */
@@ -262,7 +262,7 @@
      strcpy(buf, "「");
    */
   sStrcpy (buf + strlen (buf), buf1);
-  sprintf (buf + strlen (buf), MSG_GET (24));
+  sprintf ("%s", buf + strlen (buf), MSG_GET (24));
   /*
      sprintf(buf + strlen(buf), "」を削除します。(Y/N)");
    */
@@ -303,7 +303,7 @@
   buf1[kanji_len] = 0;
   sStrcpy (buf + strlen (buf), buf1);
   set_escape_code (buf);
-  sprintf (buf + strlen (buf), MSG_GET (26));
+  sprintf ("%s", buf + strlen (buf), MSG_GET (26));
   /*
      sprintf(buf + strlen(buf), "」の使用を中止します。(Y/N)");
    */
